.github/workflows/new_package.yml

```diff
@@ -25,8 +25,7 @@ jobs:
           # The Python script prints the package name so that we can easily
           # echo it to $GITHUB_ENV. Alternatively we could parse if with a
           # regular string from the JSON string
-          echo '${{ steps.issue-parser.outputs.jsonString }}' > pkg.json
-          pkg_name=$(./scripts/utils/create_package_template_from_json.py --json_file pkg.json)
+          pkg_name=$(./scripts/utils/create_package_template_from_json.py '${{ steps.issue-parser.outputs.jsonString }}')
```

Comment:
```suggestion
          pkg_name=$(./scripts/utils/create_package_template_from_json.py --json_string '${{ steps.issue-parser.outputs.jsonString }}')
```

Reply:
We should escape the JSON chars, maybe via `jq`? See https://stackoverflow.com/questions/10053678/escaping-characters-in-bash-for-json

Reply:
Alternatively, deleting the created file should work?!

Reply:
yes, we could delete the file after using it.

Reply:
I think `jq` is not installed. Maybe using https://github.com/sergeysova/jq-action